src/plugins/render/nodes/draw.cpp

```diff
@@ -112,9 +104,9 @@ std::set<std::string> getInputs(GLuint prog) {
 
 struct Drawable : public possumwood::Drawable {
 	Drawable(dependency_graph::Values&& vals) : possumwood::Drawable(std::move(vals)), m_vao(0) {
-		m_timeChangedConnection = possumwood::App::instance().onTimeChanged([this](float t) {
-				refresh();
-			});
+		m_timeChangedConnection = possumwood::App::instance().onTimeChanged([](float t) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'm_timeChangedConnection' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=4929755)